src/cgame/cg_ents.cpp

```diff
@@ -32,7 +32,7 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 over unused entities. It can be zero before the first frame starts adding
 entities, as none is added yet and none has to be removed yet. This should
 never reach the MAX_GENTITIES value. */
-int cg_highestActiveEntity = 0;
+size_t cg_highestActiveEntity = 0;
```

Comment:
Let's keep this int as entity numbers are always stored as ints.

Reply:
Done.